include/zephyr/bluetooth/hci_types.h

```diff
@@ -60,6 +60,7 @@ struct bt_hci_sco_hdr {
 struct bt_hci_evt_hdr {
 	uint8_t  evt;
 	uint8_t  len;
+	uint8_t  data[];
```

Comment:
```suggestion
	uint8_t  data[0];
```

Reply:
Sorry, this was a brainfart on my side - I switched around which one is the correct way (I had assumed that `hci_types.h` had already been "fixed", which contains a lot of `[0]` still)